### ISSUE TITLE
Any change for a multi line menu?

### ISSUE THREAD

**Issue by kmgilbert100** [cronvel/terminal-kit#8]
Title says it all is there any chance a multi line menu will be included in this package?

**Comment by cronvel** [comment]
By multi-line menu, do you mean a drop-down menu?

There is one, but it is still in alpha version. It's part of the "Document Model" currently under development.

You can see it working by running the script located in `sample/document/drop-down-menu-test.js`.

There is no documentation on that for instance, because the API may change during the development process.

**Comment by kmgilbert100** [comment]
I just want to replace the npm module terminal-menu as it has node 0.12*> issues, Ive had to switch to inquirer  JS ... Built a unlimited level menu system for a terminal application with inquirer ... haha .. would be cool to have a feature like that in terminal-kit... Can't belive there is no multi-level menu module out there for node... I may make one myself.

**Comment by cronvel** [comment]
Okey, I have checked 'terminal-menu'.

This feature exists in terminal-kit, in alpha version, part of the "Document Model", and the sample code is here:
sample/document/column-menu-test.js

tl;dr:

```js
var termkit = require( '../../lib/termkit.js' ) ;
var term = termkit.terminal ;

var document = term.createDocument() ;
var columnMenu = termkit.ColumnMenu.create( {
    parent: document ,
    x: 0 , // x and
    y: 5 , // y position of the top-left corner
    items: [
        {
            content: 'File' , // menu item text
            value: 'file' // menu item value (passed to the 'submit' event)
        } ,
        ... // more menu item
    ]
} ) ;

columnMenu.on( 'submit' , function( value ) {
    // submit code
} ) ;

document.giveFocusTo( columnMenu ) ;  // give the focus to the menu now
```

**Comment by kmgilbert100** [comment]
Very cool - Thanks I'll take a look at the example when I can!

**Comment by cronvel** [comment]
@kmgilbert100 This feature now exists out of the document model: [.singleColumnMenu()](https://github.com/cronvel/terminal-kit/blob/master/doc/high-level.md#ref.singleColumnMenu)
